app/src/test/java/com/invisibleteam/goinvisible/mvvm/edition/EditActivityTest.java

```diff
@@ -108,6 +108,19 @@ public void whenOptionItemIsSelectedAndTagsAreNotChanged_OnlyBackToImageActivity
     }
 
     @Test
+    public void whenClearAllTagsIsCalled_OnlyBackToImageActivityIsCalled() {
+        //Given
+        MenuItem menuItem = mock(MenuItem.class);
+        when(menuItem.getItemId()).thenReturn(2);
```

Comment:
I think that using static field from EditActivity (visibility protected for test) would be better than just plain integer value.